contrib/kafka-humio-gateway/go.mod

```diff
@@ -0,0 +1,8 @@
+module main
+
+go 1.16
```

Comment:
I wonder if we can bump this to 1.17 given velociraptor itself depends on go 1.17. That's not a deal breaker but just if we want to keep the two of them in sync. Feel free to disregard.

Reply:
The 0.6.3 branch depends on go 1.16.